framework/dbt/basic/macros/overrides.sql

```diff
@@ -1,116 +0,0 @@
-  {%- set unlogged = config.get('unlogged', default=false) -%}
-  {%- set sql_header = config.get('sql_header', none) -%}
-
-  {{ sql_header if sql_header is not none }}
-
-  create  table {{ relation }}
-  as (
-    {{ sql|replace('"crate".', "") }}
-  );
-{%- endmacro %}
-
-{% macro postgres__drop_schema(relation) -%}
-  {% if relation.database -%}
-    {{ adapter.verify_database(relation.database) }}
-  {%- endif -%}
-  {%- call statement('drop_schema') -%}
-    /* schemas are not dropped in CrateDB */
-  {%- endcall -%}
-{% endmacro %}
-
-{% macro default__drop_relation(relation) -%}
-  {% call statement('drop_relation', auto_begin=False) -%}
-    drop {{ relation.type }} if exists "{{ relation.schema }}"."{{ relation.identifier }}"
-  {%- endcall %}
-{% endmacro %}
-
-{% macro default__drop_schema(relation) -%}
-  {%- call statement('drop_schema') -%}
-    /* schemas are not dropped in CrateDB */
-  {% endcall %}
-{% endmacro %}
-
-{% macro default__create_view_as(relation, sql) -%}
-  {%- set sql_header = config.get('sql_header', none) -%}
-
-  {{ sql_header if sql_header is not none }}
-  create view "{{ relation.schema }}"."{{ relation.identifier }}" as 
-    {{ sql|replace('"crate".', "") }}
-  ;
-{%- endmacro %}
-
-{% macro postgres__rename_relation(from_relation, to_relation) -%}
-	{% do drop_relation(to_relation) %}
-	{% set schema_query = "SELECT table_type FROM information_schema.tables WHERE table_schema = '{}' AND table_name = '{}'".format(from_relation.schema, from_relation.identifier) %}
-	{% set results = run_query(schema_query) %}
-	{% if execute %}
-		{% set results_list = results.columns[0].values() %}
-	{% else %}
-		{% set results_list = [] %}
-	{% endif %}
-	{% for relation_type in results_list %}  
-		{% if relation_type == 'VIEW' %}    
-			{% set view_query = "SELECT view_definition FROM information_schema.views WHERE table_schema = '{}' AND table_name = '{}'".format(from_relation.schema, from_relation.identifier) %}
-			{% set view_definitions = run_query(view_query) %}
-			{% if execute %}
-				{% set view_definitions_list = view_definitions.columns[0].values() %}
-			{% else %}
-				{% set view_definitions_list = [] %}
-			{% endif %}
-			{% for view_definition in view_definitions_list %}  				
-				{% call statement('drop_view') -%}
-					DROP VIEW IF EXISTS {{ to_relation.schema }}.{{ to_relation.identifier }};
-				{%- endcall %}
-				{% call statement('create_view') -%}
-					CREATE VIEW {{ to_relation.schema }}.{{ to_relation.identifier }} AS {{ view_definition }}
-				{%- endcall %}
-				{% call statement('drop_view') -%}
-					DROP VIEW IF EXISTS {{ from_relation.schema }}.{{ from_relation.identifier }};
-				{%- endcall %}
-			{% endfor %}
-		{% else %}
-			{% call statement('rename_table') -%}
-				ALTER TABLE {{ from_relation.schema }}.{{ from_relation.identifier }}
-					RENAME TO {{ to_relation.identifier }}
-			{%- endcall %}
-		{% endif %}
-	{% endfor %}
-{% endmacro %}
-
```

Comment:
dbt's `rename_relation` macro is being explored over at https://github.com/crate/dbt-cratedb2/commit/3017744f649e429dfaecd3d96cfbdfc217ef28d2. Because it seems to work well, as demonstrated by a software test case, it looks like it doesn't need a macro override?

Reply:
Agreed, yes, I commented on https://github.com/crate/dbt-cratedb2/pull/2